heron/tracker/src/python/handlers/containerfilehandler.py

```diff
@@ -33,11 +34,12 @@ def get(self):
       cluster = self.get_argument_cluster()
       environ = self.get_argument_environ()
       topology_name = self.get_argument_topology()
+      role = self.get_argument(constants.PARAM_ROLE, default=None)
```

Comment:
why other parameters have very specific method but not this one?

is there any method like `self.get_argumemt_role()`?